app/main.py

```diff
@@ -1 +1,53 @@
-#  write your code here
+class Animal:
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True
+                 ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        self.is_hungry = not self.is_hungry
+        if self.is_hungry is False:
+            print(f"Eating {self.appetite} food points...")
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+    def __init__(self,
+                 name: str,
+                 appetite: int = 3,
+                 is_hungry: bool = True
+                 ) -> None:
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/209564461-afdc7993-a4a8-4265-a5d9-b80122ce69f1.png)
fix this

Reply:
also, fix the identation here